lua/nvim_lsp/dartls.lua

```diff
@@ -0,0 +1,42 @@
+local nvim_lsp = require 'nvim_lsp'
+local configs = require 'nvim_lsp/configs'
+local lsp = vim.lsp
+
+local server_name = "dartls"
+local bin_name = "dart"
+local dart_sdk_path = os.getenv("DART_SDK")
```

Comment:
I think the "DART_SDK" environment variable is not standard.
According to [official documentation](https://dart.dev/get-dart), that is not set.
If that is often used by Dart developper, would you show me a reference?

Reply:
You are right. It's not a standard. I agree with Vim variable.

Reply:
It looks like it was neccessary few years ago.. (https://github.com/dart-lang/pub/issues/715). Now it's not required.

Reply:
> It looks like it was neccessary few years ago.. (dart-lang/pub#715). Now it's not required.

`local dart_sdk_path = os.getenv("DART_SDK")` remains as it is, why not fix it?

Reply:
I am working on it... But it will (for me) take a while for that i started scripting in vim/lua like a week ago. :D
Just need find out how to pass vim variable into this lua config.

Reply:
OK, I'll put stuff later.

Reply:
@GenesisTMS Sorry for late replying.
Would you try [this PR](https://github.com/GenesisTMS/nvim-lsp/pull/1)?
And if it works well, please merge it :)

Reply:
Works very well. _Didn't try at Windows_
Thank you